doc/build/glossary.rst

```diff
@@ -1158,16 +1158,17 @@ Glossary
         values as they are not included otherwise (but note any series of columns
         or SQL expressions can be placed into RETURNING, not just default-value columns).
 
-        The backends that currently support
-        RETURNING or a similar construct are PostgreSQL, SQL Server, Oracle,
-        and Firebird.    The PostgreSQL and Firebird implementations are generally
-        full featured, whereas the implementations of SQL Server and Oracle
-        have caveats. On SQL Server, the clause is known as "OUTPUT INSERTED"
-        for INSERT and UPDATE statements and "OUTPUT DELETED" for DELETE statements;
-        the key caveat is that triggers are not supported in conjunction with this
-        keyword.  On Oracle, it is known as "RETURNING...INTO", and requires that the
-        value be placed into an OUT parameter, meaning not only is the syntax awkward,
-        but it can also only be used for one row at a time.
+        The backends that currently support RETURNING or a similar construct
+        are PostgreSQL, SQL Server, Oracle Database, and Firebird.  The
```

Comment:
**Michael Bayer** (zzzeek) wrote:

oh, we dont have a firebird backend anymore....  separate issue

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5589

Reply:
**Federico Caselli** (CaselIT) wrote:

there are still some firebird and infomix reference that were surfaced by this change

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5589